README.md

```diff
@@ -3,11 +3,31 @@
 TODO write something meaningful here
 
 ## Development
+#### Prerequisites
+- Setup the [rust toolchain](https://www.rust-lang.org/tools/install)
+```rustup default stable```
```

Comment:
The project actually depends on nightly features so these steps are needed especially if you don't plan on using nix 

- rustup toolchain install nightly
- rustup default nightly (or do it on a per project basis and not globally)
- rustup target add wasm32-unknown-unknown